ci/travis/ci.sh

```diff
@@ -157,7 +159,10 @@ test_python() {
       -python/ray/tests:test_stress_sharded  # timeout
       -python/ray/tests:test_webui
     )
-    bazel test -k --config=ci --test_timeout=600 --build_tests_only -- "${args[@]}";
+  fi
+  if [ 0 -lt "${#args[@]}" ]; then
+    install_ray
+    bazel test -k --config=ci --test_timeout=600 --build_tests_only --test_env=PYTHONPATH="${PYTHONPATH-}${pathsep}${WORKSPACE_DIR}/python/ray/pickle5_files" -- "${args[@]}";
```

Comment:
Can you explain and document why this needs to be added to the PYTHONPATH?

Reply:
Sure I'll add a comment. I think it has to do with how we do `pip install -e` rather than a normal `pip install`.